Map/RoomBase.gd

```diff
@@ -29,8 +26,4 @@ func increase_light_level() -> void:
 
 func set_torch_active() -> void:
 	has_torch = true
-	if light_level == Enums.LightLevel.LIT:
-		light_level = Enums.LightLevel.BRIGHTLY_LIT
-	else:
-		light_level = Enums.LightLevel.LIT
-	on_light_level_changed.emit()
+	increase_light_level()
```

Comment:
and if the room is unlit it will become DIMLY_LIT if we look at the enum of lights level, which is not what we want when setting a torch

Reply:
Same thing I found out with https://github.com/Saplings-Projects/1M_sub/pull/70#discussion_r1501738357